tests/foreach/foreach.t

```diff
@@ -0,0 +1,95 @@
+SETUP PYTHON
+
+$ [ -n "$PYTHON" ] || PYTHON="`which python`"
```

Comment:
Plain `python` is Python 2 pretty much everywhere except on Arch Linux, which is wrong here, as quark uses Python 3. Besides, you don't need this - just invoke quark directly as an executable script, the shebang will do its stuff correctly.